ui/overflow_menu.js

```diff
@@ -106,7 +106,7 @@ shaka.ui.OverflowMenu = class extends shaka.ui.Element {
     /** @private {ResizeObserver} */
     this.resizeObserver_ = null;
 
-    const resize = () => this.computeMaxHeight_();
+  const resize = () => this.computeOverFlowMenuPos_();
```

Comment:
Wrong indention, the linter is going to complain.

Reply:
yes I spotted it now. How can I run the linter locally so that I can fix the issues before commit and push